setup.py

```diff
@@ -24,7 +24,6 @@ def has_environment_marker_platform_impl_support():
     'cssselect>=0.9.1',
     'itemloaders>=1.0.1',
     'parsel>=1.5.0',
-    'PyDispatcher>=2.0.5',
```

Comment:
Does it mean that if `has_environment_marker_platform_impl_support()` rerturns False, than PyDispatcher is not installed?

Reply:
Oops, right, we also need to add `'PyDispatcher>=2.0.5'` in the `else` below, nice catch! :sweat_smile:

Reply:
Nice catch, and great suggestion. I am currently unable to do this though.